generators/server/templates/src/main/java/package/config/_Constants.java

```diff
@@ -8,6 +8,7 @@
     //Regex for acceptable logins
     public static final String LOGIN_REGEX = "^[_'.@A-Za-z0-9-]*$";
     // Spring profile for development and production, see http://jhipster.github.io/profiles/
+    public static final String SPRING_PROFILE_TEST = "test";
```

Comment:
I'm also going to rename `rdmport` to `random-port` as I didn't understand what it meant at first sight